tests/compile_testw3emc_wcoss.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
This file should be removed.

Reply:
I will remove this file once the testw3emc completed.    This script will be used for compile for debugging the code testw3emc